src/main/drivers/accgyro_spi_mpu6500.c

```diff
@@ -163,8 +167,32 @@ static void mpu6500AccRead(int16_t *accData)
     accData[Z] = (int16_t)((buf[4] << 8) | buf[5]);
 }
 
+#ifdef COLIBRI_RACE
+static void mpu6500SpiInit(void)
```

Comment:
The problem with this approach is that after the code is added and working no one has an incentive to improve it. Copying bad code just makes more bad code, it doesn't make it better. Your desire to add the board is totally understandable, but the community will benefit more from doing that properly and tries to force you to put more work into it before accepting the burden of supporting that code forever. (: